### PR TITLE
convert test data to open office format

### PR DESCRIPTION
**Pull request by Eric-Sommer**
### What problem do you want to solve?

Existing test cases are stored in Excel files, which creates problems when edited in LibreOffice/OpenOffice. To this end, all test data are now stored in as `ods` files.

addresses #31.

One could extend this PR by automatically convert them into `csv` in order to track changes.

### CODE REVIEW

File: .azure-pipelines.yaml [hunk 1]
```diff
@@ -21,7 +21,10 @@ jobs:
   - bash: echo "##vso[task.prependpath]$CONDA/bin"
     displayName: Add conda to PATH.
 
-  - bash: conda update conda --yes --quiet
```
Comment (MaxBlesch) [inline .azure-pipelines.yaml]: Why is that necessary? @tobiasraabe!

Comment (hmgaudecker) [reply .azure-pipelines.yaml]: @MaxBlesch : https://github.com/conda/conda/issues/9337

Comment (MaxBlesch) [reply .azure-pipelines.yaml]: Ok thanks!
